apps/src/constants.js

```diff
@@ -40,6 +40,9 @@ exports.TestResults = {
   EMPTY_FUNCTIONAL_BLOCK: 18,    // There's a functional block with an open input
   EXAMPLE_FAILED: 19,            // One of our examples didn't match the definition
 
+  // start using negative values, since we consider >= 20 to be "solved"
+  EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
```

Comment:
In addition to changing code, I think we also have a lot of db entries that would now be wrong.